src/Tangible.sol

```diff
@@ -144,24 +189,95 @@ contract Tangible is BaseTokenizedStrategy, SolidlySwapper, HealthCheck {
     function _harvestAndReport()
         internal
         override
+        checkHealth
```

Comment:
this is weird. You are adding a modifier but strategy writer still needs to call `_executeHealthCheck`?
let's talk in chat